docker/config_template.yaml

```diff
@@ -252,19 +252,19 @@ kafka:
     clusters:
         test:
             brokers:
-                - {{ default .Env.KAFKA_SEEDS "" }}:9092
+                - {{ default .Env.KAFKA_SEEDS "" }}:{{ default .Env.KAFKA_PORT "9092" }}
     topics:
-        cadence-visibility-dev:
+        {{ default .Env.VISIBILITY_NAME "cadence-visibility-dev" }}:
             cluster: test
-        cadence-visibility-dev-dlq:
+        {{ default .Env.VISIBILITY_NAME "cadence-visibility-dev" }}-dlq:
             cluster: test
     applications:
         visibility:
-            topic: cadence-visibility-dev
-            dlq-topic: cadence-visibility-dev-dlq
+            topic: {{ default .Env.VISIBILITY_NAME "cadence-visibility-dev" }}
+            dlq-topic: {{ default .Env.VISIBILITY_NAME "cadence-visibility-dev" }}-dlq
 
 publicClient:
-    hostPort: {{ default .Env.BIND_ON_IP "127.0.0.1" }}:7933
+    hostPort: {{ default .Env.BIND_ON_IP "127.0.0.1" }}:{{ default .Env.FRONTEND_PORT "7933" }}
```

Comment:
`BIND_ON_IP` won't work(and it's already broken here for some optional feature like XDC/batcher).
It should be frontend service address, like https://github.com/banzaicloud/banzai-charts/blob/cd719b684baae9e0d32785435205710f8ee32682/cadence/templates/server-configmap.yaml#L163

 Can you help change this to a new option? like `.Env.FRONTEND_SERVICE`

Reply:
Sure, I fixed and updated other `.Env.BIND_ON_IP` also.
plz, check new commit.

Reply:
>  updated other .Env.BIND_ON_IP also.

This is not correct. don't update others. 
Sorry it's a bit complicated. 
`BIND_ON_IP` is correct for `ringpop` configuration. While  in `publicClient` and `clusterInfomation`, we need to put the address of Frontend service.